config/protractor/protractor-dev.conf.js

```diff
@@ -31,8 +31,7 @@ let config = {
           url,
           common.tmp
         ]))
-        .then(() => common.exec(`npm`, [`i`, '--only=prod'], common.cwdOpts))
-        .then(() => common.exec(`npm`, [`i`, `../`], common.cwdOpts))
```

Comment:
I believe by removing the `npm install ../` line, we are essentially testing the version of builder that's already been released to npm.  That line is what installs the local version of builder.